### PR TITLE
Add help context for the find/replace overlay

### PR DESCRIPTION
**Pull request by Wittmaxi**
Adds a help context for the find/replace overlay which links to the help context for the dialog.

related to
https://github.com/eclipse-platform/eclipse.platform.ui/pull/2045

### CODE REVIEW

**Comment by Wittmaxi** [comment]
grafik

File: eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml [hunk 1]
```diff
@@ -510,6 +510,10 @@
         <description>Dialog to find and replace specified text.</description>
 		<topic label="Find/Replace with regular expressions" href="reference/ref-regex-find-replace.htm" />
     </context>
+    <context id="find_replace_overlay_context">
+        <description>Overlay to find and replace specified text.</description>
+		<topic label="Find/Replace with regular expressions" href="reference/ref-regex-find-replace.htm" />
```
Comment (HeikoKlare) [inline eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml:515]: Please correct indentation (and maybe also fix the one in the context where this was copied from).

Comment (Wittmaxi) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml:515]: Can you please check that the indentation is wrong in the IDE as well? Sometimes github shows some indentation wrong. This is what I see in the IDE:

grafik

Comment (HeikoKlare) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml:515]: The indentation is inconsistent, which is why different viewers render different visible indentations. One line uses whitespaces and one uses tabs. In the IDE, by accident, the number of whitespaces aligns with the tab size (you could even tell the IDE to render tabs differently and would then also see a difference):
image

Comment (Wittmaxi) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml]: Sorry, I didn't see that!
I have changed *all* tabs into four spaces

Comment (HeikoKlare) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml]: Thanks. Unfortunately, just transforming tabs into whitespaces is not correct, as there have been combinations of whitespaces and tabs which still behaves like a single tab (e.g. 1 whitespace + 1 tab behaves like 1 tab). So there are now places with too many whitespaces.

Comment (Wittmaxi) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml]: It looks like Eclipse has no autoformat for XML by default (??!)
I copied the file into vscode (which also doesn't autoformat XMLs by default?! I used the RedHat XML extension) and formatted it there.

Comment (HeikoKlare) [reply eclipse.platform.common/bundles/org.eclipse.platform.doc.user/contexts_Workbench.xml]: Yes, if I am not mistaken, both require (additional) extensions to (auto-)format XMLs.

**Pull request merged**